submissions/yaripey/dom-task/js/main.js

```diff
@@ -1,37 +1,101 @@
+    ]
+  },
+  pb: {
+    img: "img/pb.png",
+    text: [
+      "Princess Bonnibel Bubblegum (also called Bonnie or PB, occasionally P-bubs), is a fictional character in the American animated television series Adventure Time, created by Pendleton Ward. She is voiced by Hynden Walch.Princess Bubblegum is a \"Gum Golem\", a being composed of both human and candy DNA, and rules the Candy Kingdom, which is inhabited by other \"Candy People\". She is good friends with the series' protagonists Finn the Human and Jake the Dog, first appearing with them in Ward's 2008 pilot.",
+      "Bubblegum is typically very kind and compassionate, having a strong sense of justice, though she shows a dark side when provoked. She is highly intelligent, being responsible for the creation of the Candy People as well as various devices including a miniature time machine. She is fluent in German and Korean, using the latter language to communicate with her friend Lady Rainicorn. Bubblegum is skeptical of magic, refusing to acknowledge it as anything other than “scientific principles presented like mystical hoodoo”.",
+      "Bubblegum is highly protective of the Candy People and cares deeply for their safety, but is secretly strained by the pressures of ruling and expresses a desire for freedom. Following a near-death experience at the end of the second season, she starts to isolate herself and becomes overprotective of the Candy People, even exhibiting authoritarian tendencies temporarily in the fifth season. At the end of the sixth season, Bubblegum is deposed as ruler following an election, where she realizes she has made the Candy People too unintelligent. Following the miniseries Stakes, Bubblegum is reinstated as ruler and becomes less overprotective."
+    ]
+  }
 }
 
 document.addEventListener('DOMContentLoaded', () => {
-  let currentTab = document.querySelector('#about')
-  const mainElem = document.querySelector('main')
   const toggler = document.querySelector('.toggle')
 
-  const changeText = ({ target }) => {
-    if (target.id) {
-      if (target.id !== currentTab.id) {
-        currentTab.classList.toggle('focused')
-        mainElem.innerHTML = texts[target.id]
-        currentTab = target
-        currentTab.classList.toggle('focused')
+  // Function for changing text in main
+  const changeText = (textPart) => {
+    const currentMain = document.querySelector('main');
+
+    if (currentMain.classList[0] !== textPart) { // if this page isn't open already
+      const newElem = document.createElement('main')
+      newElem.setAttribute('class', textPart)
+
+      const newImage = document.createElement('img')
+      newImage.setAttribute('src', texts[textPart].img)
+      newImage.setAttribute('class', 'picture')
```

Comment:
try to use [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for prepare markup